pkg/api/types.go

```diff
@@ -513,7 +513,8 @@ type EnvVar struct {
 	// Required: This must be a C_IDENTIFIER.
 	Name string `json:"name"`
 	// Optional: no more than one of the following may be specified.
-	// Optional: Defaults to "".
+	// Optional: Defaults to ""; variable references ("$(VAR_NAME)") are
```

Comment:
And it would be even better if you could briefly explain the behavior rather than only referring to a spec that the reader has to search for